### ISSUE TITLE
Language based on browser settings.

### ISSUE THREAD

**Issue by santisiri** [DemocracyEarth/sovereign#320]
Take cue from browser settings to pick default language of the app.

**Comment by paulamlb** [comment]
I'm about to start making two video tutorials for vnzla and mexico. If theres any chance this can be prioritized so I can used the spanish versions as a base for the tutorials ping me.

**Comment by santisiri** [comment]
@paulamlb if you can find someone that can help us finishing the translation that would be great.

https://github.com/DemocracyEarth/sovereign/blob/master/i18n/es.i18n.json

**Comment by paulamlb** [comment]
@santisiri that list looks complete. is there another link?

**Comment by santisiri** [comment]
yes, here's the complete version: https://github.com/DemocracyEarth/sovereign/blob/master/i18n/en.i18n.json

the idea is to precisely complete the one in spanish.

**Comment by paulamlb** [comment]
I've had these translations ready in excel format for the past two weeks but cant get them in json. Asked two coders already that could not figure out properly and I couldn't figure out atom editor on my own to do it. If someone wants to do it, [here it is].(https://docs.google.com/spreadsheets/d/1jDyNQipU9R6dsnbt5wUaXOESs5i6_RX5dWhK0JWKr_8/edit?usp=sharing).

**Comment by virgile-dev** [comment]
@paulamlb let's figure out how to use translate.comunes.org it's user interface for technical translations. Here is the link for spanish : https://translate.comunes.org/projects/democracy-earth/sovereign/es/

Happy to give you a hand, here is some docs on how to use it : https://docs.weblate.org/en/weblate-2.9/index.html
